src/components/Table/Table/TableConsts.ts

```diff
@@ -5,4 +5,9 @@ export enum RowSizes {
   LARGE = "large"
 }
 
-export type RowSize = typeof RowSizes | number;
+export const RowHeights = {
+  [RowSizes.MEDIUM]: 40,
+  [RowSizes.LARGE]: 48
+};
+
+export type RowSize = typeof RowSizes;
```

Comment:
is this really needed? we don't support number anyway and I guess it was here at the first place because of that, otherwise we can just use typeof RowSizes on where RowSize is used

we can add support for number later if needed

Reply:
Good catch, removing

Reply:
Turns out it's not even used